runtime/src/comm/ofi/comm-ofi-internal.h

```diff
@@ -126,6 +127,15 @@ char* chpl_comm_ofi_dbg_prefix(void);
 
 #define CHK_SYS_CALLOC(p, n) CHK_SYS_CALLOC_SZ(p, n, sizeof(*(p)))
 
+#define CHK_SYS_POSIX_MEMALIGN(p, a, s)                                 \
+    do {                                                                \
+        if ((posix_memalign(&(p), (a), (s))) != 0) {                    \
```

Comment:
Use `sys_memalign` instead and rename macro?

Reply:
Will do.  I'm not even sure why I didn't do that in the first place!

Reply:
It'll fix the CI failure, too.